sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -148,7 +148,7 @@ using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
 ### Recognize Content
 Recognize text and table data, along with their bounding box coordinates, from documents.
 
-```C# Snippet:FormRecognizerSample3RecognizeContent
+```C# Snippet:FormRecognizerSample2RecognizeContentFromUri
```

Comment:
My understanding is we want to highlight the methods that take input from file over the ones taken from a URI since these are more common, so I suggest using the file methods for these readme samples

Reply:
does this apply to the readme too? we have 1 sample from file and the rest from URI

Reply:
will leave as it is for now. created issue to revisit here: https://github.com/Azure/azure-sdk-for-net/issues/11508

Reply:
yeah, i'm confused about your question because I left the comment on the README

Reply:
LOL. I'm sorry :( you are absolutely right.

Reply:
I did URI because it's less code.  I think it's fine either way, as long as a sample exists for both approaches somewhere.